infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -83,6 +83,10 @@ public class LogReplicationSinkManager implements DataReceiver {
     // Current topologyConfigId, used to drop out of date messages.
     private long topologyConfigId = 0;
 
+    // Cluster id of the remote cluster from which this Sink Manager receives
+    // updates
+    private String remoteClusterId = null;
```

Comment:
what is this used for?

Reply:
Since we now have a SinkManager corresponding to each Source, it is used in the name of the thread created in this class - 
https://github.com/CorfuDB/CorfuDB/blob/1c34029a603f35bebd37264ea65a44738fed8250/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java#L165